### ISSUE TITLE
Dev User Registration - Administrative Developer

### ISSUE THREAD

**Issue by seobrand** [egortimofeev/genius#5]
Accept query string from ClickFunnel and populate the initial signup screen correctly

**Comment by egortimofeev** [comment]
Please give more details what I need to accept and where I can get the details about it.

**Comment by egortimofeev** [comment]
seems done. waiting on feedback from Jenelle on #7. closing this for now as it is duplicate of #7 in this case.
